src/tests/algorithms_test/calorimetry_CalorimeterHitDigi.cc

```diff
@@ -29,6 +29,18 @@ TEST_CASE( "the clustering algorithm runs", "[CalorimeterHitDigi]" ) {
 
   auto detector = dd4hep::Detector::make_unique("");
 
+  auto& serviceSvc = algorithms::ServiceSvc::instance();
+  [[maybe_unused]] auto& geoSvc = algorithms::GeoSvc::instance();
+  serviceSvc.setInit<algorithms::GeoSvc>([&detector](auto&& g) {
+    g.init(detector.get());
+  });
+  [[maybe_unused]] auto& randomSvc = algorithms::RandomSvc::instance();
+  serviceSvc.setInit<algorithms::RandomSvc>([](auto&& r) {
+    r.setProperty("seed", static_cast<size_t>(1));
+    r.init();
+  });
```

Comment:
```suggestion
  [[maybe_unused]] auto& randomSvc = algorithms::RandomSvc::instance();
  auto seed = Catch::getSeed();
  serviceSvc.setInit<algorithms::RandomSvc>([seed](auto&& r) {
    r.setProperty("seed", static_cast<size_t>(seed));
    r.init();
  });
```

Reply:
Also needs `#include <catch2/catch_get_random_seed.hpp>` above the current context.

Reply:
After 3.1.1, https://github.com/catchorg/Catch2/commit/dc001fa935d71b4b77f263fce405c9dbdfcbfe28

Reply:
Requiring 3.1.1, released Oct 19, 2022. Upgrading nightly to include.

Reply:
Unfortunately not finding any defines with the version in catch2 to make this work with pre-3.1.1 too...